orttraining/orttraining/python/training/onnxblock/checkpoint_utils.py

```diff
@@ -17,3 +20,16 @@ def save_checkpoint(parameters, path_to_checkpoint):
     trainable_params = [param.SerializeToString() for param in trainable_params]
     non_trainable_params = [param.SerializeToString() for param in non_trainable_params]
     _internal_save_checkpoint(trainable_params, non_trainable_params, path_to_checkpoint)
+
+
+def load_checkpoint(path_to_checkpoint, model):
```

Comment:
I think let's call this function something that indicates that the checkpoint is loaded on the model. We might want to add `load_checkpoint` in the future that returns the loaded checkpoint state.